docs/considerations.rst

```diff
@@ -0,0 +1,214 @@
+##########################
+Considerations for ME-fMRI
+##########################
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. 
+
+The possible costs and benefits of multi-echo fMRI
+==================================================
+The following are a few points to consider when deciding whether or not to collect multi-echo data.
+
+Possible increase in TR
+-----------------------
+The one difference with multi-echo is a slight time cost.
+For multi-echo fMRI, the shortest echo time (TE) is essentially free since it is collected in the
```

Comment:
Should we reference Molly bright's method, here ? I think a lot of folks come to tedana thinking they only need two echos.